### ISSUE TITLE
Ссылка с привязкой ко времени

### ISSUE THREAD

**Issue by npats2007** [max-frai/book-audio#27]
Хорошо цитировать\делиться "текстом". Функционал типа ютуба "копировать URL с привязкой ко времени". В соцсети готовую ссылку заряжать удобно.

**Comment by npats2007** [comment]
https://youtu.be/kSVHRFKvH3w?t=161

**Comment by max-frai** [comment]
Нужно подумать.
А что если у человека на этой книге свой прогресс (совсем другой), а он переходит по вашей ссылке. Сбрасывать ли его текущий прогресс прослушивания? Он может не подозревать об этом. Если не сбрасывать, то получается слишком сложная логика. А как потом обратно переключиться на "свой" прогресс?

**Comment by npats2007** [comment]
Да, тут неоднозначно. В URL использовать признак "привязка ко времени"? Если с ним начали проигрывать книгу, то не запоминать новое место проигрывателя. При любом действии на сервисе сбрасывать признак "привязка ко времени" и работать в штатном режиме.
Считаю небольшой вероятность встретить ссылку на прослушиваемую в данный момент мною книгу. Даже если "цитатой" собьется место прослушивания потеря будет небольшой. Надо взвесить стоимость функционала/компромисс.
